uiowa/uiowa
# iFrame Title Filter module investigation

Resolves #3393 


# How to test
1. `blt ds --site=default`
2. `drush cim`
3. On a page, embed Youtube, Google maps, Instagram, Candid Career, Panopto, etc. and see if you can see a <title> attribute for each one.
